CondTools/Ecal/python/copyBadStrip_cfg.py

```diff
@@ -39,11 +40,12 @@
     IsDestDbCheckedInQueryLog=cms.untracked.bool(True),
     SinceAppendMode=cms.bool(True),
     Source=cms.PSet(
-     firstRun = cms.string('120000'),
+     firstRun = cms.string('200000'),
      lastRun = cms.string('10000000'),
      OnlineDBSID = cms.string('cms_omds_lb'),
-     OnlineDBUser = cms.string('cms_ecal_conf'),
-     OnlineDBPassword = cms.string('*************'),
+#     OnlineDBSID = cms.string('cms_orcon_adg'),  test on lxplus
+     OnlineDBUser = cms.string('cms_ecal_r'),
+     OnlineDBPassword = cms.string('3c4l_r34d3r'),
```

Comment:
@depasse I don't think putting the ecal account password in clear in the release code is good practice